examples/interactions/src/jvmMain/kotlin/RequestResponseExample.kt

```diff
@@ -15,28 +15,25 @@
  */
 
 import io.rsocket.kotlin.*
-import io.rsocket.kotlin.connection.*
+import io.rsocket.kotlin.core.*
 import io.rsocket.kotlin.payload.*
+import io.rsocket.kotlin.transport.local.*
 import kotlinx.coroutines.*
 
 fun main(): Unit = runBlocking {
-    val (clientConnection, serverConnection) = SimpleLocalConnection()
-
-    launch {
-        serverConnection.startServer {
-            RSocketRequestHandler {
-                requestResponse = {
-                    val data = it.data.readText()
-                    val metadata = it.metadata?.readText()
-                    println("Server received payload: data=$data, metadata=$metadata")
-
-                    Payload(data.repeat(3), "SERVER_METADATA") //response
-                }
+    val server = LocalServer()
+    RSocketServer().bind(server) {
```

Comment:
This change is much nicer, feels more kotlin-y, particularly `requestResponse {` instead of `requestResponse = {`